datafusion/core/tests/physical_optimizer/enforce_sorting.rs

```diff
@@ -983,7 +983,7 @@ async fn test_soft_hard_requirements_with_multiple_soft_requirements_and_output_
     ));
 
     let expected_input = [
-        "OutputRequirementExec",
+        "OutputRequirementExec: order_by=[([non_nullable_col@1], asc)], dist_by=SinglePartition",
```

Comment:
It seems that we can remove the extra `[]` around order by expr to make the display concise.

`order_by=[(non_nullable_col@1, asc)]`

Reply:
yeah, agreed.